### ISSUE TITLE
Duplicate PAI-TF logs

### ISSUE THREAD

**Issue by Yancey0623** [sql-machine-learning/sqlflow#1569]
The sql program:
``` sql
SELECT *
FROM alifin_jtest_dev.sqlflow_test_iris_train
TO TRAIN DNNClassifier
WITH
    model.n_classes = 3,
	model.hidden_units = [10, 20],
	train.num_workers=2,
	train.num_ps=2,
	train.save_checkpoints_steps=20,
	train.epoch=10,
	train.batch_size=4,
	train.verbose=2
COLUMN sepal_length, sepal_width, petal_length, petal_width
LABEL class
INTO my_dnn_model;
```

Print the duplicate logs:

``` text
2019/12/30 07:48:13 CREATE TABLE CmFOyVVGg5YEyINN AS SELECT *
FROM alifin_jtest_dev.sqlflow_test_iris_train
2019/12/30 07:48:16 CREATE TABLE XZQTmcov7ZvDZZIx AS SELECT *
FROM alifin_jtest_dev.sqlflow_test_iris_train
saving model to: oss://sqlflow-models/my_dnn_model/?role_arn=acs:ram::1694276552806203:role/paitftrain&host=cn-zhangjiakou.oss-internal.aliyun-inc.com
```

**Comment by typhoonzero** [comment]
That's not a duplication actually, those two sql statements are used for create temp table for training set and validation set.
